prisma/migrations/20220729101258_init/migration.sql

```diff
@@ -11,9 +11,9 @@ CREATE TABLE "User" (
     "id" TEXT NOT NULL,
     "name" VARCHAR(128) NOT NULL,
     "email" VARCHAR(64) NOT NULL,
-    "phoneNumber" VARCHAR(32) NOT NULL,
```

Comment:
@ShouryaMup-CAW Please check with @pichu07 on this and come to a common migration and schema. Use NOT NULL wherever required.